tex/gloss-odia.ldf

```diff
@@ -0,0 +1,157 @@
+% Language definition file
+%
+% Translations provided by ଅଶୋକ ଦାସ୍ <das@pas.rochester.edu> and ଅଭିଷେକ ଜେନା <avisekjena@gmail.com>
+% TODO implement Odia calendar
+
+\ProvidesFile{gloss-odia.ldf}[polyglossia: module for Odia]
+
+\RequirePackage{devanagaridigits}
+\RequirePackage{odiadigits}
+
+\PolyglossiaSetup{odia}{
+  bcp47=or,
+  %bcp47-language=or,
+  %bcp47-script=Odia,
+  script=Odia,
+  scripttag=odia,
+  langtag=ODI,
+  hyphennames={odia},
+  hyphenmins={2,2},%CHECK
+  fontsetup=true,
+  localnumeral=odianumerals
+  %TODO nouppercase=true,
+}
+
+% BCP-47 compliant aliases
+\setlanguagealias*{odia}{or}
+
+
+%Adapted from gloss-bengali.ldf 
+
+\newif\ifodia@devanagari@numerals
+\newif\ifodia@odia@numerals
+\define@choicekey*+{odia}{numerals}[\xpg@val\xpg@nr]{Devanagari,Odia,Western}[Devanagari]{%
+   \ifcase\xpg@nr\relax
+      % Devanagari:
+      \odia@odia@numeralsfalse%
+      \odia@devanagari@numeralstrue%
```

Comment:
Strangely, this was not there in the reference ldf that I used. 
Although I have added this line, I am not very sure of its use / impact.

Do guide me if I need to do any additional tests.

Reply:
Yes, this is not strictly necessary, and some ldf files are missing the u specifier. But since Odia has two different scripts, it is better to indicate that.